pyposeidon/utils/cast.py

```diff
@@ -285,6 +285,9 @@ def run(self, **kwargs):
         self.rdate = self.model.rdate
         ppath = self.ppath
 
+        ppath = pathlib.Path(ppath).resolve()
+        ppath = str(ppath)
```

Comment:
if `self.ppath` is already a str, then converting to `pathlib.Path` and then casting back to a str should not be necessary. We could probably just use `ppath = os.path.realpath(ppath)` (not tested).

https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module

Reply:
The tests pass, so I will use `os.path.realpath` and see how it goes.